ui/src/app/components/delete-persistent-disk-button.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+
+import { LinkButton } from 'app/components/buttons';
+import { styles } from 'app/components/runtime-configuration-panel/styles';
+
+interface DeletePersistentDiskProps {
+  onClick: () => void;
+  style?: React.CSSProperties;
+}
+
+export const DeletePersistentDiskButton = ({
```

Comment:
I didn't add any unit tests for this component because I don't think there are any valuable tests to add. I would be testing things like "onClick is passed to onClick" or "renders a LinkButton" which I think is self-evident in declarative components. Relying on the type validations and manual tests seems like enough to me.

I could add an "it renders" smoke test, if that would be valuable.

(Side note: I suspect snapshot testing would be a great fit here, but I think it's low-priority to explore that class of tests today)